.github/scripts/run-docker.sh

```diff
@@ -11,7 +11,6 @@ CONFIG_DIR=${PWD}/score/configs/${CONFIG_VER}
 CONFIG_ENV=${CONFIG_DIR}/config-${CONFIG_VER}.env
 # Use the latest pytorch-benchmark image
 TORCH_IMAGE_ID=torchbench/pytorch-benchmark:latest
-NO_TURBO_FILE="/sys/devices/system/cpu/intel_pstate/no_turbo"
```

Comment:
i'm confused- no-turbo setting here seemed to work before; why did it start having a permission problem now that we moved to metal?

if anything i'd expect the machine_util would have problems with permissions since it runs _inside_ docker?  can you explain better in the git commit message?

Reply:
It didn't have permission problem before because on x5large this file doesn't exist.
Inside docker, I have tested that machine_util.py can *read* all the system values, but it will not be able to write into system files (Read-only file system). Therefore, the check_machine_configured() function still works.

There is another problem with the Docker environment: Docker provides a ('Ubuntu', '18.04', 'bionic') distribution (which PyTorch official docker uses), not Amazon Linux.